### ISSUE TITLE
BERT returns different embedding for same sentence

### ISSUE THREAD

**Issue by rshah1990** [google-research/bert#857]
I am using pre-trained BERT for creating features, for same sentence it produces different result in two different runs. Do we have to set some random state to produce consistent result?  I am using **pytorch-transformers** for reading pre-trained model.

**Comment by rshah1990** [comment]
as mentioned I am using pytorch-transformers to read pre-trained BERT model , I have not set model in evaluation mode so because of dropout layer i was getting slightly different embedding for same text. 
I am closing this issue.
